### DISCUSSION TITLE
Too many epochs

### DISCUSSION THREAD

**Discussion by sakatomo38** [voicepaw/so-vits-svc-fork#788] [Q&A]
I am completely new in AI learning, and installed everything I need today.

I have 2,200 <10sec audio files to learn, and set my batch size as much as I could, which is 14.
However it seems to be 9,999 epochs. It takes 2.5 minutes to finish 1 epoch, which virtually takes forever.

So I asked ChatGPT about this, and what it says is I do not need to wait that much since most of the software stops earlier.
Is this true?

Is having less audio files the only way to solve this problem?

<img width="535" alt="image" src="https://github.com/voicepaw/so-vits-svc-fork/assets/137661755/ccb1ab11-e4d3-4ae8-8843-f3e82ea4dea1">
<img width="477" alt="image" src="https://github.com/voicepaw/so-vits-svc-fork/assets/137661755/9708a83b-ca49-41a8-95f2-733fabc903d1">

**Comment by 34j** [comment]
Basically a single epoch is sufficient
